Source/Chatbook/Models.wl

```diff
@@ -0,0 +1,47 @@
+BeginPackage["Wolfram`Chatbook`Models`"]
+
+Needs["Wolfram`Chatbook`"]
+Needs["GeneralUtilities`" -> None]
+
+
+Begin["`Private`"]
+
+
+Options[SetModel]={"SetLLMEvaluator"->True}
+
+SetModel[args___]:=Catch[setModel[args],"SetModel"]
+
+setModel[model_,opts:OptionsPattern[SetModel]]:=setModel[$FrontEndSession,model,opts]
+
+setModel[scope_,model_Association,opts:OptionsPattern[SetModel]]:=(
+  Needs["GeneralUtilities`"];
+  If[TrueQ[OptionValue["SetLLMEvaluator"]],
+  	System`$LLMEvaluator=System`LLMConfiguration[System`$LLMEvaluator,model];
+  ];
+  CurrentValue[scope, TaggingRules] = 
+  	GeneralUtilities`ToAssociations @CurrentValue[$FrontEndSession, TaggingRules];
```

Comment:
This isn't the best place to fix tagging rules that have been converted to lists. This method should only really be used at the level of `$FrontEnd`, otherwise it will cause all values in a higher scope to be hard coded in. For example, if there are global preferences set:
```
In[1]:= CurrentValue[$FrontEnd, {TaggingRules, "ChatNotebookSettings"}]

Out[1]= <|"ToolOptions" -> <|"WolframLanguageEvaluator" -> <|"AllowedReadPaths" -> All|>|>, "Assistance" -> False, "LLMEvaluator" -> "Birdnardo"|>
```

They are inherited at the notebook level:
```
In[2]:= CurrentValue[nbo = CreateNotebook[], {TaggingRules, "ChatNotebookSettings"}]

Out[2]= <|"ToolOptions" -> <|"WolframLanguageEvaluator" -> <|"AllowedReadPaths" -> All|>|>, "Assistance" -> False, "LLMEvaluator" -> "Birdnardo"|>
```

So this will cause those values to get hardcoded into the notebook options:
```
In[3]:= CurrentValue[nbo, TaggingRules] = GeneralUtilities`ToAssociations[CurrentValue[nbo, TaggingRules]];

In[4]:= Options[nbo, TaggingRules]

Out[4]= {TaggingRules -> <|"ChatNotebookSettings" -> <|"ToolOptions" -> <|"WolframLanguageEvaluator" -> <|"AllowedReadPaths" -> All|>|>, "Assistance" -> False, "LLMEvaluator" -> "Birdnardo"|>|>}
```

Reply:
I took ok the tagging rules fix